app/services/db-export/export-data-files.service.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const path = require('path')
+
+const fs = require('fs').promises
+const os = require('os')
+
+/**
+ * Export converted data to a temporary file
+ * @module ExportDataFilesService
+*/
+
+/**
+ * Converts the provided data to CSV format using the ConvertToCsvService and writes it to a file
+ *
+ * @param {Object} data The data to be converted to CSV and written to the file
+ *
+ * @returns {Boolean} True if the file is written successfully and false if not
+ */
+async function go (data) {
+  try {
+    await fs.writeFile(_filenameWithPath('Billing Charge Categories Table Export.csv'), data)
```

Comment:
I wouldn't put spaces in a filename. I'd write it more like this: `billing_charge_categories_table_export.csv`. Personally I think it's a good habit to get into. What do you guys think @Cruikshanks @StuAA78 

Sorry @Beckyrose200

Reply:
<img src="https://media1.giphy.com/media/NEvPzZ8bd1V4Y/giphy.gif"/>
Changes made, thank you :D

Reply:
💯